torchfix/torchfix.py

```diff
@@ -25,18 +25,56 @@
 
 DISABLED_BY_DEFAULT = ["TOR3", "TOR4"]
 
-
-def GET_ALL_VISITORS():
+def GET_ALL_VISITOR_CLS():
```

Comment:
As we don't call the visitors here anymore, should be just 

ALL_VISITORS_CLS = [
        TorchDeprecatedSymbolsVisitor,
        TorchRequireGradVisitor,
        TorchSynchronizedDataLoaderVisitor,
        TorchVisionDeprecatedPretrainedVisitor,
        TorchVisionDeprecatedToTensorVisitor,
        TorchUnsafeLoadVisitor,
        TorchReentrantCheckpointVisitor,
]